token-swap/program/src/curve.rs

```diff
@@ -50,6 +50,16 @@ impl Default for SwapCurve {
     }
 }
 
+/// Clone takes advantage of pack / unpack to get around the difficulty of
```

Comment:
Not sure if this is too hacky -- it's mainly used for testing purposes

Reply:
Is it used exclusively in tests? If so, perhaps rename or at least comment to that effect.

Reply:
I'll put in a comment and `#[cfg(test)]` to make it as clear as possible